plugins/electron.js

```diff
@@ -4,16 +4,23 @@ const _                 = require('lodash')
 const Promise           = require('bluebird')
 const { readFileAsync } = Promise.promisifyAll(require('fs'))
 const { join }          = require('path')
+const YAML              = require('yamljs')
 
 module.exports = {
   name: 'electron',
   description: 'Checks for the presence of electron in the package.json dependencies',
   test: path => {
     return readFileAsync(join(path, 'package.json'))
     .then(str => {
-      if (_.get(JSON.parse(str), 'dependencies.electron', false)) return true
-
-      return false
+      const pkg = JSON.parse(str)
+      if (_.get(pkg, 'dependencies.electron', false)) return true
+      if (_.get(pkg, 'devDependencies.electron', false)) return true
```

Comment:
Keep in mind there might be some edge cases that require electron but are not electron apps, like when you install NightmareJS to do web scraping or automated testing, and configure it with your own Electron version, but they are not very common, so its fine, but it keep in mind if we hit issues later

Reply:
@jviotti good point, I inverted the order of the checks so if a repo.type different than `electron` is set we ignore the dependencies.